project/AkkaBuild.scala

```diff
@@ -160,7 +160,7 @@ object AkkaBuild {
     licenses := Seq(("Apache-2.0", url("https://www.apache.org/licenses/LICENSE-2.0.html"))),
     homepage := Some(url("https://akka.io/")),
     description := "Akka is a toolkit for building highly concurrent, distributed, and resilient message-driven applications for Java and Scala.",
-
+    scmInfo := Some(ScmInfo(url("https://github.com/akka/akka"), "git@github.com:akka/akka.git")),
```

Comment:
I don't think I usually include `.git` - can't hurt though

Reply:
I just copied from another place where I knew it worked.
It was previously (in M7):
```
    <scm>
        <url>https://github.com/akka/akka</url>
        <connection>scm:git:https://github.com/akka/akka.git</connection>
        <developerConnection>scm:git:git@github.com:akka/akka.git</developerConnection>
    </scm>
```

We can define it to be the same as before.